week-1/interpret/percentage-change.js

```diff
@@ -11,10 +11,10 @@ console.log(`The percentage change is ${percentageChange}`);
 
 // Read the code and then answer the questions below
 
-// a) How many function calls are there in this file? Write down all the lines where a function call is made
+// a) How many function calls are there in this file? Write down all the lines where a function call is made - line 4,5,8,10
 
-// b) Identify all the lines that are variable reassignment statements
+// b) Identify all the lines that are variable reassignment statements - line 4,5, 
 
-// c) Identify all the lines that are variable declarations
+// c) Identify all the lines that are variable declarations - line 1,2
```

Comment:
There are 2 more variable declarations in the code. They are variables whose value cannot change. What may they be?

Reply:
line 7 and 8 because they are const variable